src/megaapi_impl.cpp

```diff
@@ -18427,30 +18571,39 @@ void MegaApiImpl::sendPendingRequests()
                     break;
                 }
 
-                // encode the MegaStringMap as a TLV container
-                TLVstore tlv;
-                string value;
-                const char *buf, *key;
-                MegaStringList *keys = stringMap->getKeys();
-                for (int i=0; i < keys->size(); i++)
+                if (type == ATTR_ALIAS)
                 {
-                    key = keys->get(i);
-                    buf = stringMap->get(key);
-
-                    size_t len = strlen(buf)/4*3+3;
-                    value.resize(len);
-                    value.resize(Base64::atob(buf, (byte *)value.data(), int(len)));
-
-                    tlv.set(key, value);
+                    // always get updated value before update it
+                    const char *uh = getMyUserHandle();
```

Comment:
We have now the `MegaClient::uid` that doesn't need to convert `me` into B64 and delete later on. Better to use that one :)